woocommerce/woocommerce-android
# [Mobile Payments] With TTP visible, onboarding error hides bottom rows on the payments hub

Closes: #8766


### Description

The PR:
* ~~Removes "learn more" from the payments hub. In order to make it work right it has to be moved inside of the Recycler View, and I personally don't think that it brings much to the table. The payments hub contains many things so it is not really clear what a user should "learn more" about. Context is missing here @AnirudhBhat  @samiuelson wdyt about that?~~ **Added Learn More as an item of the Recycler View**
* Adds padding so onboarding error doesn't hide last rows
* Fixes determination of "the same item" and "the same content" so there is no more blinking of the "manage card reader" row

### Testing instructions

* Open payments hub with onboarding completed. Notice that the "manage card reader" row didn't blink and there is small padding at the bottom
* Open payments hub with onboarding not completed. Notice that the "manage card reader" row didn't blink and there is  padding at the bottom enough to show all the rows behind the error pop up

### Images/gif


#### Before

https://user-images.githubusercontent.com/4923871/230606786-79f6c017-e0b2-402a-a3e8-488893ef53e1.mp4

#### After

https://user-images.githubusercontent.com/4923871/230606791-ed505de1-ab2c-47b1-8697-5f806d1dfd95.mp4
